pepper-apis/src/main/java/org/broadinstitute/ddp/constants/RouteConstants.java

```diff
@@ -174,6 +174,7 @@
         public static final String EMAIL  = "email";
         public static final String SUCCESS  = "success";
         public static final String UMBRELLA = "umbrella";
+        public static final String UMBRELLA_GUID = "umbrellaGuid";
```

Comment:
If we're renaming it, maybe remove the `UMBRELLA` above since that's the old one?

Reply:
Why do we need to rename this at all? The parameter is already called `umbrella`- the description can make it clear we're looking for the GUID, although I don't think it's as big a deal since we're using GUIDs as UIDs everywhere else.